tests/reporter/kubernetes.go

```diff
@@ -100,32 +100,44 @@ func (r *KubernetesReporter) Dump(duration time.Duration) {
 		return
 	}
 
+	nodesDir := r.createNodesDir()
+	podsDir := r.createPodsDir()
+
 	duration += 5 * time.Second
 	since := time.Now().Add(-duration)
 
+	nodes := getNodeList(virtCli)
+	nodesWithVirtLauncher := getNodesWithVirtLauncher(virtCli)
+	pods := getPodList(virtCli)
+	vmis := getVMIList(virtCli)
+
 	r.logClusterOverview()
 	r.logEvents(virtCli, since)
 	r.logNamespaces(virtCli)
-	r.logNodes(virtCli)
 	r.logPVCs(virtCli)
 	r.logPVs(virtCli)
-	r.logPods(virtCli)
 	r.logAPIServices(virtCli)
 	r.logServices(virtCli)
 	r.logEndpoints(virtCli)
-	r.logVMIs(virtCli)
 	r.logConfigMaps(virtCli)
 	r.logSecrets(virtCli)
-	r.logAuditLogs(virtCli, since)
-	r.logDMESG(virtCli, since)
-	r.logJournal(virtCli, duration, "")
-	r.logJournal(virtCli, duration, "kubelet")
-	r.logVMs(virtCli)
-	r.logDomainXMLs(virtCli)
-	r.logLogs(virtCli, since)
-	r.logSRIOVInfo(virtCli)
 	r.logNetworkAttachmentDefinitionInfo(virtCli)
```

Comment:
maybe NAD should be near SRIOV under "network stuff" ? not sure...

Reply:
nad can exists even without sriov, and fits more to the general cluster resources imo